src/software/ai/hl/stp/play/hardware_challenge_plays/BUILD

```diff
@@ -4,125 +4,127 @@ package(default_visibility = ["//visibility:public"])
 # "factory" design pattern to work are linked in
 # https://www.bfilipek.com/2018/02/static-vars-static-lib.html
 
-cc_library(
-    name = "scoring_with_static_defenders_play",
-    srcs = ["scoring_with_static_defenders_play.cpp"],
-    hdrs = ["scoring_with_static_defenders_play.h"],
-    deps = [
-        "//shared:constants",
-        "//software/ai/hl/stp/play",
-        "//software/ai/hl/stp/tactic/move:move_tactic",
-        "//software/logger",
-        "//software/util/generic_factory",
-    ],
-    alwayslink = True,
-)
+# TODO: Update challenge plays to use new Skills architecture
```

Comment:
we can probably delete the challenge plays.

these were a robocup challenge a while ago

Reply:
done